01_fib/anzranjana/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func Fib(number int) {
+	firstNum, nextNum := int64(0), int64(1)
+	switch number {
+	case 0:
+		fmt.Fprintln(out, number)
+	default:
+		if number < 0 { // Negative Fibonacci series
```

Comment:
The logic here looks pretty similar for both the positive and negative Fibonacci numbers, can you abstract this out somehow to avoid duplication?